lib/env.ts

```diff
@@ -1,4 +1,7 @@
-type EnvKey = 'SPID_SESSION_TOKEN';
+export type EnvKey = (
```

Comment:
I suggest to have this as a [io-ts](https://github.com/gcanti/io-ts) type so you can deserialize the needed vars using

`EnvKey.decode(process.env)`

Reply:
Is this the expected solution?

```
export const get = (key: EnvKey): Option<string> =>
  EnvKey.decode(process.env).fold(
    _ => none,
    v => some(v)
  );
```